SignalRServiceBenchmarkPlugin/utils/Commander/Program.cs

```diff
@@ -0,0 +1,278 @@
+﻿using CommandLine;
+using Common;
+using Medallion.Shell;
+using Renci.SshNet;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+// TODO: dispose commands
+namespace Commander
+{
+    class Program
+    {
+        private static string _baseName = "publish";
+
+        public static void Main(string[] args)
+        {
+            var argsOption = ParseArgs(args);
+
+            Util.CreateLogger(argsOption.LogDirectory, argsOption.LogName, argsOption.LogTarget);
+
+            // Publish dlls
+            var appserverExecutable = PubishExcutable(argsOption.AppserverProject, _baseName);
+            var masterExecutable = PubishExcutable(argsOption.MasterProject, _baseName);
+            var slaveExecutable = PubishExcutable(argsOption.SlaveProject, _baseName);
+
+            //// DEBUG
+            //var baseName = "publish";
+            //var appserverExecutable = new FileInfo(Path.Combine(argsOption.AppserverProject, $"{baseName}.zip"));
+            //var masterExecutable = new FileInfo(Path.Combine(argsOption.MasterProject, $"{baseName}.zip"));
+            //var slaveExecutable = new FileInfo(Path.Combine(argsOption.SlaveProject, $"{baseName}.zip"));
+
+            // Copy executables and configurations
+            CopyExcutables(argsOption.Username, argsOption.Password,
```

Comment:
There are too many arguments in this function. Please pass "argsOption" instead.

Reply:
I refactor the code, please take a look.